### DISCUSSION TITLE
How Do I Use This?

### DISCUSSION THREAD

**Discussion by andrewdwallo** [konnco/filament-import#22] [Q&A] [answered]
I notice that all of the files I upload are going to my local storage in my app, but how do I append this data into my model?
I have a Transactions Model for a csv bank statement but when I upload the file it says ERROR "Undefined array key "" ??
How do I make it to where All the data will append to my Form I have in the "TransactionResource.php" class?

**Comment by frankyso** [comment, marked as answer]
Hi @andrewdwallo can you please inform us how to reproduce the issue?

**Reply by andrewdwallo** [reply]
I figured it out... I was using something outdated. Your plugin is amazing btw. I hope you continue to update this...

**Reply by frankyso** [reply]
okay then, thank you

**Reply by sergioraldana** [reply]
Leaving a blank field in match selectors throws Undefined array key ""

**Reply by frankyso** [reply]
okay will fix, thanks for the information

**Reply by andrewdwallo** [reply]
Yeah actually just happened to me again. I thought it wasn't a problem at first but you have to fill out every field even if the field isn't required... I need to have fields that are optional as well.

**Reply by frankyso** [reply]
I have fixed the current issue. please have a check

**Reply by andrewdwallo** [reply]
It works! thank you
